Fix/documentation installation (#1087)
esm-tools/esm_tools: docs/yaml.rst

  command ``export`` is not needed here, just define the variable as
  ``VAR_NAME: VAR_VALUE`` or as a nested dictionary.

Syntax
^^^^^^

> ``add_export_vars`` can be defined as a list of exports or as a dictionary. We are strongly encourage to define the export variables as a dictionary.

Reviewer: ```suggestion
``add_export_vars`` can be defined as a dictionary or a list of strings (list of strings will be deprecated from version 7.0 onwards). We are strongly encourage to define the export variables as a dictionary.
```

As list
'''''''

> ::

Reviewer: ```suggestion
.. code-block:: yaml
```
This is the best way of highlighting `yaml` files in `rst`

    [ ... ]

        add_export_vars:
            - "NETCDF_DIR=/sw/dataformats/netcdf/intel.18/4.7.3/skl/"
            - "LD_LIBRARY_PATH=$NETCDF_DIR/lib/:$LD_LIBRARY_PATH"
            - "NETCDF_CXX_LIBRARIES=$NETCDF_DIR/lib"

    [ ... ]

As dictionary
'''''''''''''

> ::

Reviewer: ```suggestion
.. code-block:: yaml
```

    [ ... ]

        add_export_vars:
            NETCDF_DIR: /sw/dataformats/netcdf/intel.18/4.7.3/skl/
            LD_LIBRARY_PATH: '$NETCDF_DIR/lib/:$LD_LIBRARY_PATH'
        

>     [ ... ]

Reviewer: Please, put this one first :)
